saleor/graphql/webhook/subscription_types.py

```diff
@@ -1209,11 +1212,11 @@ def resolve_order(root, info: ResolveInfo):
 
 class FulfillmentTrackingNumberUpdated(SubscriptionObjectType, FulfillmentBase):
     class Meta:
-        doc_category = DOC_CATEGORY_ORDERS
-        root_type = "Fulfillment"
-        enable_dry_run = True
+        root_type = None
+        enable_dry_run = False
```

Comment:
Here we can actually enable dry dun as there is only one object in the subscription context.